pyqt-apps/siriushla/widgets/line_edit.py

```diff
@@ -31,3 +32,15 @@ def update_format_string(self):
             validator = QRegExpValidator(re, self)
             self.setValidator(validator)
         super().update_format_string()
+
+
+class SiriusWaveformLineEdit(SiriusLineEdit):
+    """Line Edit with bugfix for waveform setpoints."""
+
+    def value_changed(self, value):
+        """Force value to be a numpy array."""
+        super().value_changed(
+            np.array([int(value)], dtype=int)
```

Comment:
why forcing the value to be an integer? If this is the case, maybe the name of the class is not appropriate.

Reply:
fixed! thanks @fernandohds564 !